docs/configuration.md

```diff
@@ -130,7 +130,7 @@ conf/remote-shuffle-conf.yaml file.
 
 | Key | Value Type | Default Value | Version | Required | Description |
 | --- | ---------- | ------------- | ------- | -------- | ----------- |
-| `remote-shuffle.cluster.id` | String | `/default-cluster` | 1.0.0 | false | The unique ID of the remote shuffle cluster used by high-availability. Different shuffle clusters sharing the same zookeeper instance must be configured with different cluster id. This config must be consistent between the `ShuffleManager` and `ShuffleWorker`s. |
+| `remote-shuffle.cluster.id` | String | `/default-cluster` | 1.0.0 | false | The unique ID of the remote shuffle cluster used by high-availability. It must start with '/'. Different shuffle clusters sharing the same zookeeper instance must be configured with different cluster id. This config must be consistent between the `ShuffleManager` and `ShuffleWorker`s. At job side, the ID is treated as a name prefix, which means a job can only use the shuffle cluster whose ID starts with the configured value. For example, if a shuffle cluster is configured with cluster ID 'ab', then the job configured with cluster ID 'a' or 'ab' can use it, but the job configured with cluster ID 'b' or 'ac' can't use it. |
```

Comment:
Hi @wsry, I have a question here, why we need to treat this as the name prefix instead of choosing the cluster when it's exact same ?

Reply:
Prefix is superset of exact matching. If you make the prefix just the same with the cluster name, then it is exact matching. Besides, prefix has other advantages, for example, when upgrading, the jobs can select the new cluster with the same prefix, the jobs already running can keeping using the old cluster, and after all jobs finish, we can safely stop the old cluster, which means that upgrading does not influence running jobs, so it can be a way to achieve user-imperceptible upgrading.

Reply:
Thanks for your explanation. I have went through the PR,  Nice improvement 👍🏻 , Looking forward to be merged. 

One more question here, do we need to add an extra config option for the flink job eg: `remote-shuffle.cluster-id.prefix` and do not share the `remote-shuffle.cluster.id` So that we can know it is the prefix of the remote shuffler cluster's cluster id by name.

Reply:
I think using a different config option is good suggestion which make it more clear. I will update the PR.